pkg/cmd/ssh_aws.go

```diff
@@ -63,15 +63,15 @@ func sshToAWSNode(nodeName, path, user, pathSSKeypair string, sshPublicKey []byt
 	fmt.Println("Data fetched from target shoot cluster.")
 	fmt.Println("")
 
-	fmt.Println("(2/4) Setting up bastion host security group")
+	fmt.Println("(2/4) Creating bastion host")
+	a.createBastionHostInstance()
```

Comment:
inside of createBastionHostInstance,  it requre the `a.BastionSecurityGroupID` however it created at steps 3/4  
https://github.com/gardener/gardenctl/blob/c58b2124d44340f2e22fc5a4b4ae969d34c55840/pkg/cmd/ssh_aws.go#L260

so which causes the `ssh` failure.